specification/summary-care-record.yaml

```diff
@@ -38,43 +38,49 @@ paths:
       responses:
         "200":
           description: OK successful
           content:
             application/fhir+json:
               schema:
                 type: object
+              example: "{\n    \"OperationOutcome\": {\n      \"issue\": {\n        \"severity\": {\n            \"value\": \"information\"\n        },\n        \"code\": {\n            \"value\": \"informational\"\n        },\n        \"diagnostics\": {\n            \"value\": \"Resource has been successfully updated.\"\n        }\n      }\n    }\n}"
+        "400":
```

Comment:
This 400 response is the same as the 412?

Reply:
yes correct, please see diagnostic value for error (link to enums)